app/guide/attendance.md

```diff
@@ -14,10 +14,8 @@ To register children as attending or absent:
 1. Go to **Sessions**.
 2. Go to the **Today** tab.
 3. Click on the name of the school you’re attending.
-4. Click on **Register attendance**.
+4. Click on **Review** under **Register attendance**.
```

Comment:
I’d have this click the tab rather than the review link:

```suggestion
4. Click on **Register attendance**.
```

Reply:
We might also want to mention on this page that you can only register children when a session is in progress.